nistats/first_level_model.py

```diff
@@ -311,17 +312,17 @@ def fit(self, run_imgs, events=None, confounds=None,
             expected per run_img. Ignored in case designs is not None.
             If string, then a path to a csv file is expected.
 
-        confounds: pandas Dataframe or string or list of pandas DataFrames or
+        regressors: pandas Dataframe or string or list of pandas DataFrames or
                    strings,
-            Each column in a DataFrame corresponds to a confound variable
+            Each column in a DataFrame corresponds to a regressor variable
             to be included in the regression model of the respective run_img.
             The number of rows must match the number of volumes in the
             respective run_img. Ignored in case designs is not None.
             If string, then a path to a csv file is expected.
 
         design_matrices: pandas DataFrame or list of pandas DataFrames,
             Design matrices that will be used to fit the GLM. If given it
-            takes precedence over events and confounds.
+            takes precedence over events and regressors.
```

Comment:
Does it really need to be a confound ? For instance for ASL we can include additionnal regressors by convolving the PRF with the conditions. May be the terminology covariate can be used here ?